### ISSUE TITLE
wav2vec 2.0 pretraining does't converge

### ISSUE THREAD

**Issue by jinggaizi** [pytorch/fairseq#3144]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
first , i modify the transformer config to ffn2048-embed512 to reduce parameters, i pretraing on my own datasets(800h) with max-update 400000 and the final accuracy is 0,62, it seems like normal. However, when i pretraing on another datasets(700h)，it does't converge, the accuracy curve is as follows
![image](https://user-images.githubusercontent.com/8711081/104996269-8df6f900-5a62-11eb-8499-c43a4d203e81.png)

i try to adjust the batchsize and learning to 1/2 ，it's not going better

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by alexeib** [comment]
not much i can do to help since i have no idea what dataset you are using or how exactly you modified the architecture (besides changing dimensionalities). is the new dataset still 16khz audio?
